src/classes/BDI_DataImportService.cls

```diff
@@ -41,16 +41,20 @@ global with sharing class BDI_DataImportService {
     public static final String FM_HELP_TEXT = 'Help Text';
     public static final String FM_DATA_IMPORT_FIELD_MAPPING = 'Data Import Field Mapping';
 
-    private BDI_FieldMapping fieldMapper;
+    public BDI_FieldMapping fieldMapper;
```

Comment:
Just curious, what was the motivation to switch this to public?

Reply:
It needs to be accessible to the Additional object service for parsing out the tiers.